.github/workflows/docs.yml

```diff
@@ -3,7 +3,17 @@ name: Build docs
 on:
   push:
     branches: [master]
+    paths-ignore:
+      - "tests/**"
+      - "docker/**"
+      - "assets/**"
+      - ".github/workflows/**"
```

Comment:
Let's focus on `pull_request`, since we want to test our docs on `push` to master in all cases

```suggestion
```